py/desitarget/test/test_mock_fluctuations.py

```diff
@@ -1,73 +0,0 @@
-import os
-import unittest
-import numpy as np
-from astropy.table import Table
-
-from desitarget.mock import fluctuations
-from desitarget import desi_mask as Mx
-from desitarget import obsconditions
-
-
-class TestFluctuations(unittest.TestCase):
```

Comment:
I agree that we should test what we can. This test was removed because the actual implementation was completely different.